src/magnify/registry.py

```diff
@@ -85,8 +840,221 @@ def pc_chip(**kwargs):
     return pipe
 
 
-def mrbles(**kwargs):
-    pipe = Pipeline("read", config=kwargs)
+def mrbles(
+    spectra: str | PathLike | StringIO,
+    codes: str | PathLike | StringIO,
+    data: ArrayLike | str,
+    debug: bool = False,
+    times: Sequence[int] | None = None,
+    channels: Sequence[str] | None = None,
+    flatfield: float = 1.0,
+    darkfield: float = 0.0,
+    overlap: int = 102,
+    min_bead_radius: int = 5,
+    max_bead_radius: int = 25,
+    low_edge_quantile: float = 0.1,
+    high_edge_quantile: float = 0.9,
+    num_iter: int = 5000000,
+    min_roundness: float = 0.3,
+    roi_length: int = 61,
+    search_channel: str | list[str] | None = None,
+    reference: str = "eu",
+    squeeze: bool = True,
+    roi_only: bool = False,
+    drop_tiles: bool = True,
+) -> xr.Dataset | list[xr.Dataset]:
+    """
+    Processes image data for MRBLEs (Microsphere-based Rolling Circle Amplification Beads), applying flatfield correction, bead detection, and spectral decoding.
```

Comment:
This isn't what the acronym means it actually means microspheres with ratiometric barcode lanthanide encoding.

Reply:
Thank you ;)

Resolved